fastlane/Fastfile

```diff
@@ -29,6 +43,30 @@ before_all do
   end
 end
 
+desc 'Runs the unit tests for the project'
+lane :test do | options |
+  run_tests(
+    workspace: workspace,
+    scheme: internal_scheme,
```

Comment:
Even if that's the default configuration set up on the project, I think it's better to be explicit here:
```suggestion
    scheme: internal_scheme,
    configuration: 'Debug',
```